src/GitVersionCore/BuildAgents/GitHubActions.cs

```diff
@@ -9,9 +11,13 @@ public class GitHubActions : BuildAgentBase
 
         public GitHubActions(IEnvironment environment, ILog log) : base(environment, log)
         {
+            this.environment = environment;
         }
 
         public const string EnvironmentVariableName = "GITHUB_ACTIONS";
+        public const string GitHubSetEnvTempFileEnvironmentVariableName = "GITHUB_ENV";
+        private readonly IEnvironment environment;
```

Comment:
Perhaps `BuildAgentBase` can expose a protected `Environment` property instead?

Reply:
It already does! Missed that. Pushing a commit now.